schema/proto3/gateway.proto

```diff
@@ -19,15 +19,18 @@ syntax = "proto3";
 package alcor.schema;
 
 option java_package = "com.futurewei.alcor.schema";
-option java_outer_classname = "AuxiliaryGateway";
+option java_outer_classname = "Gateway";
 
-enum AuxGatewayType {
+enum GatewayType {
     NONE = 0;
     ZETA = 1;
+    TGW = 2;
```

Comment:
it will be great if we can put comments on what TGW/IGW/NGW stands for.

Reply:
comments added